test/utils_test.jl

```diff
@@ -0,0 +1,10 @@
+using Test, DelayEmbeddings
+
+println("\nTesting utils...")
+
+@testset "utils" begin
+
+    A = orthonormal(50, 50)
+    @test size(A) == (50, 50)
```

Comment:
Wait, the `orthonormal` function was not tested anywhere else so far...? In anycase, please also add 2 more tests, one that checks that `A isa Matrix`, and another one `B = orthonormal(3,3)` and `@test B isa SMatrix`.

Reply:
Actually, the function was neither tested nor used (anymore?) in this repository, I did a keyword search through all files and it only came up in its definition in `utils.jl`. But yes, good point, I'll write those tests in a bit once I'm at my PC